shared/chat/conversation/input.desktop.js

```diff
@@ -27,6 +28,15 @@ class Conversation extends Component<void, Props, State> {
     super(props)
     const {emojiPickerOpen} = props
     this.state = {emojiPickerOpen, text: _cachedInput[props.selectedConversation] || ''}
+    this._globalKeyDownHandler = ev => this._handleGlobalKeyPress(ev)
```

Comment:
you can just define this in the class itself, you can replace the flow type on line 20 with this basically